MonoGame.Framework/Platform/Graphics/RenderTarget2D.DirectX.cs

```diff
@@ -143,38 +138,57 @@ internal void ResolveSubresource()
             lock (GraphicsDevice._d3dContext)
             {
                 GraphicsDevice._d3dContext.ResolveSubresource(
-                    _msTexture,
+                    GetMSTexture(),
                     0,
-                    _texture,
+                    GetTexture(),
                     0,
                     SharpDXHelper.ToFormat(_format));
             }
         }
 
-        internal override void CreateTexture()
+        protected internal override Texture2DDescription GetTexture2DDescription()
         {
-            var desc = GetTexture2DDescription();
-            desc.BindFlags |= BindFlags.RenderTarget;
-            if (Mipmap)
-                desc.OptionFlags |= ResourceOptionFlags.GenerateMipMaps;
+            var desc = base.GetTexture2DDescription();
 
-            _texture = new SharpDX.Direct3D11.Texture2D(GraphicsDevice._d3dDevice, desc);
+            if (MultiSampleCount == 0)
```

Comment:
That's one thing I change here, when we use the _msTexture we don't use the base._texture as a renderTarget.